package.json

```diff
@@ -15,8 +15,9 @@
     "build": "gatsby build --prefix-paths",
     "serve-build": "gatsby serve --prefix-paths",
     "update-icons": "git submodule update --init --remote && node scripts/createIconTypes.js && node scripts/createIconNames.js",
-    "build-pdf": "python3 scripts/pdf/generate_pdf.py",
-    "build-all-pdfs": "for i in product_docs/docs/**/*/ ; do echo \"$i\"; python3 scripts/pdf/generate_pdf.py ${i%} || exit 1; done",
+    "build-pdf": "docker-compose -f docker/docker-compose.build-pdf.yaml run --rm --entrypoint scripts/pdf/generate_pdf.py pdf-builder",
```

Comment:
I don't feel too strongly one way or another on this - but there is at least one person on the docs team who is already used to building PDFs with this command. Since we've changed it to be dockerized, they'll have to change their setup to install docker etc. Just something to keep in mind

Reply:
I think we should deprecate running pandoc and wkhtmltopdf locally completely, except for power users.

I'm going to add an issue to address to get rid of needing pandoc locally as well.

Reply:
Yeah that's fair :+1: